services-translation/src/main/java/com/gfgtech/translation/dto/TranslationDto.java

```diff
@@ -0,0 +1,23 @@
+package com.gfgtech.translation.dto;
+
+import java.util.List;
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import lombok.extern.jackson.Jacksonized;
+
+/** @author Claudiu MAN */
+@Getter
+@Jacksonized
+@Builder
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TranslationDto {
+    List<TranslationPayload> payload;
+
+    public boolean isTranslationPayloadNull() {
```

Comment:
Do we really need thise methods?
If we need to have a method, I think these could be merged into one and use `CollectionUtils.isEmpty(payload)`